uploader/setup.sh

```diff
@@ -18,11 +18,16 @@ curl -o ${UPLOADER_HOME}/pantheon.py https://raw.githubusercontent.com/redhatacc
 echo "Ensure uploader script is executble"
 chmod +x ${UPLOADER_HOME}/pantheon.py 
 
-# Create a symlink to /usr/local/bin if not exists
+# Create a symlink to ~/bin/pantheon if not exists
 if [ ! -h "$PANTHEON_BIN" ]; then
   echo "Create a symlink to $PANTHEON_BIN"
-  sudo ln -s ${UPLOADER_HOME}/pantheon.py $PANTHEON_BIN
+  ln -s ${UPLOADER_HOME}/pantheon.py $PANTHEON_BIN
```

Comment:
I'm not sure it's necessary to put the uploader in `~/.pantheon` and then symlink it, instead of just putting the script itself into `~/bin`, but this doesn't hurt anything. Was there a reason you chose to do it this way?

Reply:
~/.pantheon today only has pantheon.py, in the future we can also place other helpful files in there. E.g a README with helpful info that's not covered in `pantheon -h`

Reply:
Ok, that makes sense!